src/unity/python/turicreate/visualization/plot.py

```diff
@@ -0,0 +1,29 @@
+import time
+import json
+
+class Plot(object):
+    def __init__(self):
+        self.plot_object = tc.extensions._Plot()
+
+    def show(self):
+        self.plot_object.show()
+
+    def save_vega(self, filepath, include_data=True):
+        file_contents = self.plot_object.get_vega(include_data)
+
+        with open(filepath, 'w') as fp:
+            json.dump(file_contents, fp)
+
+    def get_data(self, filepath, include_data=True):
+        return self.plot_object.get_data()
+
+    def get_vega(self, include_data=True):
+        return self.plot_object.get_vega(include_data)
+
+    def _repr_javascript_(self):
+        from IPython.core.display import display, HTML, clear_output, update_display
```

Comment:
The imports for `clear_output` and `update_display` don't seem to be used.

Reply:
removed imports